src/airdrop.ts

```diff
@@ -667,6 +722,11 @@ export class Airdrop {
       GOVERNANCE_VERIFIER_PK,
     );
 
+    const [verifierSignature, _signatureBump] = web3.PublicKey.findProgramAddressSync(
```

Comment:
Why move this?

Reply:
Move receipt and voteRecord closer to the proposal finding because of the TODO added. Will make it easier later